app/Models/Category.php

```diff
@@ -14,4 +14,12 @@ class Category extends Model
     public function game() {
         return $this->belongsToMany(Game::class);
     }
+
+    public function run()
```

Comment:
I believe you need to create an actual Factory class rather than a Category model using:

sail artisan make:factory CategoryFactory

The code you created, _I think_, is correct, but we don't need any code in the Factory class yet per the issue instructions. At least that's how I interpreted it.

Reply:
So do you think this won't work unless I make it in Git? even though there's no code in it

Reply:
correct, this run function is not needed and should be removed. use the artisan command to create a CategoryFactory class.